scanner/Makefile

```diff
@@ -451,14 +451,14 @@ e2e-deploy: db-host-path :=
 e2e-deploy: debug-templates :=
 
 # Wait timeout for the E2E deployment to be successful.
-e2e-deploy: timeout := 10m
+e2e-deploy: timeout := 30m
 
 e2e-deploy: $(e2e-chart-d) $(e2e-conf-files-t) $(e2e-certs-t)
 	@echo "+ $@" db=$(db-tag) scanner=$(scanner-tag)
 	helm $(if $(debug-templates),template,upgrade --install) --debug $(release) $< \
 	    --namespace $(namespace) \
 	    --create-namespace \
-	    $(if $(CI),--wait,--atomic) \
+	    $(if $(CI),,--atomic) \
```

Comment:
What's the reason for removing `--wait`? Is it due to the logic introduced in the `Wait deployment` job? Makes me wonder if there's a way to include that functionality for local uses of `e2e-deploy`.

Reply:
> Is it due to the logic introduced in the Wait deployment job?

Yes.

> Makes me wonder if there's a way to include that functionality for local uses of e2e-deploy.

We could, I just went ahead and decided to rely on `--atomic` when doing this locally. Specially because you likely can inspect the cluster yourself, which is not easy to do from CI.